SS14.Admin/SS14.Admin.csproj

```diff
@@ -10,6 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="CsvHelper" />
```

Comment:
You didn't add a matching entry to `Directory.Packages.props`?

Reply:
fixed